app/components/agenda/agenda-header/template.hbs

```diff
@@ -293,4 +293,20 @@
       cancelForm=(action "cancelEditSessionForm")
     }}
   {{/web-components/vl-modal}}
+{{/if}}
+{{#if isDeletingAgenda}}
+  {{#web-components/vl-modal
+    isOverlay=true
+    title=(t "agenda-delete")
+  }}
+  {{web-components/vl-loader text=(concat (t "agenda-delete-message") " " (t "please-be-patient"))}}
+  {{/web-components/vl-modal}}
+{{/if}}
+{{#if isLockingAgenda}}
+  {{#web-components/vl-modal
+    isOverlay=true
+    title=(t "agenda-lock")
+  }}
+  {{web-components/vl-loader text=(concat (t "agenda-lock-message") " " (t "please-be-patient"))}}
+  {{/web-components/vl-modal}}
```

Comment:
Hier zou ik opteren om de loading text te computen in de `component.js` file.
Iets zoals:
```
shouldShowLoader: computed('isDeletingAgenda ', 'isLockingAgenda', function() {
    return this.isDeletingAgenda || this.isLockingAgenda;
})

loaderText: computed('isDeletingAgenda ', 'isLockingAgenda', function() {
   if(thiis.isDeletingAgenda) {
     return intl...
   } 
   if(thiis.isLockingAgenda) {
     return intl...
   } 
 return "";
})
loaderTitle..
```

Dat haalt de duplicate code uit de template.